scripts/fusion-swap/fill.ts

```diff
@@ -146,6 +156,10 @@ async function main() {
     );
     return;
   }
+  orderConfig.fee.maxCancellationPremium = new BN(
+    orderConfigJson.fee.maxCancellationPremium,
```

Comment:
Why was it necessary to define only this field from the `fee` object? Maybe it can be made consistent with the others?

Reply:
This is not defining, but converting it from hex encoded string (from `orderConfigJson`) to `BN`. The rest of the fields are of `number` type and is of the same type in `orderConfigJson` already.